python/lsst/jointcal/jointcal.py

```diff
@@ -25,8 +24,11 @@
 
 __all__ = ["JointcalConfig", "JointcalTask"]
 
+Photometry = collections.namedtuple('Photometry', ('fit', 'model'))
+Astrometry = collections.namedtuple('Astrometry', ('fit', 'model', 'sky_to_tan_projection'))
 
-class JointcalRunner(pipeBase.TaskRunner):
+
+class JointcalRunner(pipeBase.ButlerInitializedTaskRunner):
```

Comment:
I cleaned up that docstring and removed the comment about multiprocessing. I don't know about pickleability, so I'll just remove that whole part of the note. I've got plans to try some multiprocessing work in jointcal for I/O.